includes/sql-schema/update.php

```diff
@@ -153,6 +153,9 @@
         $db_rev = $filename;
         if ($insert) {
             dbInsert(array('version' => $db_rev), 'dbSchema');
+            if ($db_rev >= 6) {
+                $insert = 0;
```

Comment:
We have to avoid the first 5 files as version doesn't exist in dbSchema until 006.sql.